include/pstl_offload/internal/usm_memory_replacement.h

```diff
@@ -25,13 +26,23 @@
 
 #include "usm_memory_replacement_common.h"
 
+#if _WIN64
+#    include <corecrt.h>
+#    pragma comment(lib, "pstloffload.lib")
+#endif
+
 namespace __pstl_offload
 {
 
 // allocation can be requested before static ctor or after static dtor runs, have a flag for that
 // keep it out of __offload_policy_holder_type to not access object before ctor or after dtor
 static std::atomic_bool __device_ready;
 
+// Under Windows, we must not use functions with explicit alignment for malloc replacement, as
+// an allocated memory would be released by free() replacement, that has no alignment argument.
+// Mark such allocations with special alignment. Use 0, as this is not valid alignment.
+static constexpr std::size_t __ignore_alignment = 0;
```

Comment:
I would make it `inline`. Like:
```suggestion
inline constexpr std::size_t __ignore_alignment = 0;
```

Reply:
Done.